arbnode/node.go

```diff
@@ -426,7 +426,7 @@ func ConfigAddOptions(prefix string, f *flag.FlagSet, feedInputEnable bool, feed
 	DelayedSequencerConfigAddOptions(prefix+".delayed-sequencer", f)
 	BatchPosterConfigAddOptions(prefix+".batch-poster", f)
 	f.String(prefix+".forwarding-target", ConfigDefault.ForwardingTargetImpl, "transaction forwarding target URL, or \"null\" to disable forwarding (iff not sequencer)")
-	f.Bool(prefix+".pre-check-txs", ConfigDefault.PreCheckTxs, "if true, verify basic state transition requirements of incoming RPC transactions before processing them")
+	f.String(prefix+".tx-pre-checker-mode", ConfigDefault.TxPreCheckerMode, "how strict to be when checking txs before forwarding them. \"disabled\" to allow everything, \"loose\" to allow nonce too high, or \"strict\" to deny anything failing locally")
```

Comment:
Not necessarily right now:
We should also have a mode that only rejects Txs that cannot possibly be accepted by a later block.. so reject on signature or nonce too low, but not on balance.
GasFee - perhaps only reject if under what GasFee could possibly decrease to?

Reply:
Implemented, thanks for the recommendation! I've made this the new default :)